options_compile.cpp

```diff
@@ -178,7 +178,6 @@ std::string EffectiveOptionsFilter::processOptions(const OpenCLArgList &args,
   effectiveArgs.push_back("-cl-kernel-arg-info");
   effectiveArgs.push_back("-fno-validate-pch");
   effectiveArgs.push_back("-fno-caret-diagnostics");
-  effectiveArgs.push_back("-dwarf-column-info");
```

Comment:
why this change? is this outdate code?

Reply:
Yes, with community commit https://reviews.llvm.org/rGb0b5162fc23c55906a461366f8acef2431d951c5
option -dwarf-column-info is no longer exist.

Reply:
I see, thanks